src/formEditor/FormXmlSerializer.ts

```
@@ -0,0 +1,357 @@
+  /** Переместить элемент формы (по id) в новую позицию */
+  moveElement(
+    elementId: number,
+    targetParentId: number,
+    insertBeforeId: number | null
+  ): boolean {
+    const formNode = this.findTag(this.orderedTree, 'Form');
+    if (!formNode) return false;
+
+    // Найти и удалить элемент из текущего родителя
+    const removed = this.removeElementById(formNode, elementId);
+    if (!removed) return false;
```

ESLint

[error] Expected { after 'if' condition.

(curly)